backend/app/schemas/rail_user.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel
+
+
+class RailUser(BaseModel):
+    email: str
+    api_key: str = ""
+
+
+class RailUserCreate(RailUser):
```

Comment:
This class is not used yet. Unless we want to use it in the future, we can delete this